geoalchemy2/admin/dialects/sqlite.py

```diff
@@ -16,25 +17,35 @@
 from geoalchemy2.types import _DummyGeometry
 
 
-def load_spatialite(dbapi_conn, connection_record, init_mode=None):
-    """Load SpatiaLite extension in SQLite DB.
+def load_spatialite_driver(dbapi_conn, *args):
+    """Load SpatiaLite extension in SQLite connection.
 
-    The path to the SpatiaLite module should be set in the `SPATIALITE_LIBRARY_PATH` environment
-    variable.
+    .. Warning::
+        The path to the SpatiaLite module should be set in the `SPATIALITE_LIBRARY_PATH`
+        environment variable.
 
-    The init_mode argument can be `'NONE'` to load all EPSG SRIDs, `'WGS84'` to load only the ones
-    related to WGS84 or `'EMPTY'` to not load any EPSG SRID.
-
-    .. Note::
-
-        It is possible to load other EPSG SRIDs afterwards using the `InsertEpsgSrid(srid)`.
+    Args:
+        dbapi_conn: The DBAPI connection.
     """
     if "SPATIALITE_LIBRARY_PATH" not in os.environ:
         raise RuntimeError("The SPATIALITE_LIBRARY_PATH environment variable is not set.")
     dbapi_conn.enable_load_extension(True)
     dbapi_conn.load_extension(os.environ["SPATIALITE_LIBRARY_PATH"])
     dbapi_conn.enable_load_extension(False)
 
+
+def init_spatialite(dbapi_conn, *args, init_mode: Optional[str] = None):
```

Comment:
Why adding the `*args` here?

Reply:
Because when it is called as a listener it receives another argument that we don't use.